private-templates-service/client/src/components/AdaptiveCardPanel/AdaptiveCardPanel.tsx

```diff
@@ -24,6 +25,25 @@ interface Props {
   template: Template;
 }
 
+function retrieveStateValue(template: Template): string {
+  if (template.instances && template.instances[0] && template.instances[0].state) {
+    let state = template.instances[0].state;
+    switch (state) {
```

Comment:
so a cleaner way to do this would be `return PostedTemplate.StateEnum[state]`, this returns a string for the enum value, so you can get rid of this switch statement

Reply:
to clarify, if `state = PostedTemplate.StateEnum.Draft`, `PostedTemplate.StateEnum[state] = "Draft"`

Reply:
Ooh thanks.